understanding/extended-audio-description-prerecorded.html

```diff
@@ -58,105 +56,90 @@
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.7:</span>Extended Audio Description (Prerecorded) (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.7:</span>拡張音声解説 (収録済) (レベル AAA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Where pauses in foreground audio are insufficient to allow <a href="#dfn-audio-description">audio descriptions</a> to convey the sense of the video, <a href="#dfn-extended-audio-description">extended audio description</a> is provided for all <a href="#dfn-prerecorded">prerecorded</a>
-                     <a href="#dfn-video">video</a> content in <a href="#dfn-synchronized-media">synchronized media</a>.
+                  <p>前景音の合間が、<a href="#dfn-audio-description">音声解説</a>で映像の意味を伝達するのに不十分な場合、<a href="#dfn-synchronized-media">同期したメディア</a>に含まれているすべての<a href="#dfn-prerecorded">収録済</a>の<a href="#dfn-video">映像</a>コンテンツに対して、<a href="#dfn-extended-audio-description">拡張音声解説</a>が提供されている。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Videos can be played with more detailed audio descriptions.</dd>
+                  <dt>目標</dt>
+                  <dd>より詳細な音声解説付きで映像を再生できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide extended spoken descriptions of the visual content in videos.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>映像の視覚的な内容に対して、拡張した発話音声による説明を提供する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who are blind or who cannot understand the visual content can have it described
-                     to them while playing videos.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>全盲又は視覚的な内容を理解できない人が、再生中の映像について解説してもらうことができる。
```

Comment:
「再生中の映像」となっていますが、ここの while playing videos は「ユーザーが動画を再生している最中」という意味かと思います。ユーザー自ら動画を一時停止して、動画とは別に用意された音声を聞く、といったことをしなくてよいというニュアンスを含んでいるのだろうと思います。
たとえばこんな感じではどうでしょうか?

```suggestion
                  <dd>全盲又は視覚的な内容を理解できない人が、映像を再生しながら、その内容を説明してもらえる。
```

Reply:
@bakera ありがとうございます。

> 「再生中の映像」となっていますが、ここの while playing videos は「ユーザーが動画を再生している最中」という意味かと思います。ユーザー自ら動画を一時停止して、動画とは別に用意された音声を聞く、といったことをしなくてよいというニュアンスを含んでいるのだろうと思います。

御意です！